src/editor.rs

```diff
@@ -137,7 +137,7 @@ impl<'a> Editor<'a> {
     /// # Errors
     ///
     /// Will return `Err` if an error occur when enabling termios raw mode, creating the signal hook
```

Comment:
Just noticed this typo, can you also fix it?

```suggestion
    /// Will return `Err` if an error occurs when enabling termios raw mode, creating the signal hook
```